content/news/2024-11-26-FLUX/index.md

```diff
@@ -0,0 +1,57 @@
+
+In this blog post, we'll explore the FLUX.1 models, their unique attributes, and how to use them effectively inside Galaxy.
+
+## Overview of FLUX Models
+
+**FLUX.1 [schnell]**
+* Purpose: Designed for speed.
+* License: Openly available under the Apache 2.0 license.
+* Use Case: Ideal for personal use and rapid prototyping.
+
+**FLUX.1 [dev]**
+* Purpose: Guidance-distilled model for non-commercial applications, offering near-professional quality.
+* License: Open-weight model intended for non-commercial contexts.
+* Use Case: Suited for research and non-commercial exploration.
+
+Both models provide easy access and versatility, making them excellent choices for text-to-image projects. While FLUX.1 [schnell] focuses on performance, FLUX.1 [dev] bridges the gap between quality and efficiency.
+
+## Why Use FLUX Models Inside Galaxy?
+
+Galaxy's open platform supports workflows across life sciences, data analysis, and beyond. By integrating FLUX models, users can:
+
+* Generate visualizations: Convert textual descriptions into detailed images.
+* Streamline workflows: Incorporate generative AI into multi-step Galaxy pipelines.
+* Leverage open weights: FLUX's availability under open licenses aligns with Galaxy's ethos of accessibility and reproducibility.
+
+To ensure high performance and speed, we are using a CUDA-based container to run the FLUX models on GPUs. This setup allows for fast inference times when working with the models from Hugging Face. By leveraging Hugging Face’s pre-trained models, we can efficiently load the weights and run inference directly within the Galaxy environment. The CUDA-based container ensures that the heavy computations involved in text-to-image generation are handled smoothly, providing users with a seamless experience.
+
+## Setting Up FLUX in Galaxy
+
+The FLUX models are already available on [UseGalaxy.eu](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/bgruening/black_forest_labs_flux/black_forest_labs_flux/2024+galaxy3). Simply log in to your Galaxy instance and search for the FLUX tool in the tool panel.
+
+Select the desired model (FLUX.1 [schnell] or FLUX.1 [dev]) based on your requirements.
+Choose between File or Text based input and select your txt file or enter your text prompt in the designated input field.
+Click Run Tool and let Galaxy handle the rest!
+Once the process is complete, you’ll find the generated image in your Galaxy history, ready to view, share, or use in your workflows.
+
+For an example, I used the text prompt "A galaxy full of stars" to generate an image. Here's [the result](https://usegalaxy.eu/u/brilliantarash/h/flux):
+
+Text Prompt: `A galaxy full of stars`
+
+Generated Image:
+
+![FLUX_on_text_prompt_(A_galaxy_full_of_stars)](./FLUX_on_text_prompt_(A_galaxy_full_of_stars).png)
```

Comment:
```suggestion
![FLUX_on_text_prompt_(A_galaxy_full_of_stars)](./FLUX_on_text_prompt-a_galaxy_full_of_stars.png)
```

I'm not sure brackets in filenames will not be problematic on Windows computers etc ... better to avoid them